Documentation/ceph-csi-drivers.md

```diff
@@ -92,3 +92,49 @@ kubectl create -f https://raw.githubusercontent.com/csi-addons/volume-replicatio
 2. Enable the volume replication controller:
    - For Helm deployments see the [csi.volumeReplication.enabled setting](helm-operator.md#configuration).
    - For non-Helm deployments set `CSI_ENABLE_VOLUME_REPLICATION: "true"` in operator.yaml
+
+## generic ephemeral volume support
```

Comment:
How about this simple title, then the details about generic support are in the description.
```suggestion
## Ephemeral Volumes
```